data/transactions/logic/TEAL_opcodes.md

```diff
@@ -1048,6 +1048,24 @@ pushbytes args are not added to the bytecblock during assembly processes
 
 pushint args are not added to the intcblock during assembly processes
 
+## pushbytess bytes ...
+
+- Opcode: 0x82 {varuint length} [({varuint value length} bytes), ...]
```

Comment:
How about we change the first `length` to `count`?

For consistency, we would do this for the cblock opcodes and pushints too.